src/town/a5.py

```diff
@@ -23,6 +23,9 @@ def get_wp_location(self) -> Location: return Location.A5_WP
     def can_heal(self) -> bool: return True
     def can_buy_pots(self) -> bool: return True
     def can_resurrect(self) -> bool: return True
+
+    def can_identify(self) -> bool: return True
```

Comment:
You have to add this to the interface class also (i_act.py). Whenever you have the id config set to true and you are in another act it will raise an error and fail.

Reply:
i was wondering about that. fixed :)